config/rbac/role.yaml

```diff
@@ -1,11 +1,13 @@
+
+---
```

Comment:
@paulosjca Hi Paulo, I am not sure why it was generated this way, is there any concerns of having this form?